packages/cluster/src/commands/up.js

```diff
@@ -115,24 +88,32 @@ module.exports = {
             alias: 'u',
             desc: 'Indicate that d2 cluster should re-download cached files',
             type: 'boolean',
-            default: defaults.update,
         },
         image: {
             alias: 'i',
             desc: 'Specify the Docker image to use',
             type: 'string',
-            default: defaults.image,
         },
         dhis2Version: {
             desc: 'Set the DHIS2 version',
             type: 'string',
-            default: defaults.dhis2Version,
+        },
+        dbVersion: {
+            desc: 'Set the database version',
+            type: 'string',
+        },
+        channel: {
+            desc: 'Set the release channel to use (default: stable)',
```

Comment:
Should this use the same `defaults` logic as the other properties?  So basically that would mean removing the `if !substitutes.channel` check from `makeDockerImage` and adding `channel: 'stable'` to defaults ...?

Reply:
Sure, I can change the `defaults.channel` to `stable` and use the same logic. 👍 

I cannot remove the `!substitutes.channel` check though since the `makeDockerImage` function doesn't do the configuration resolution itself, and if the function is used with a substitution object which doesn't pass in the `channel`, it won't get replaced with an empty string. Would have been a bit simpler if the stable repo was called `core-stable` and not just `core`, but I'd rather have a little bit of extra complexity here and a nicer repo name than the other way around.

Reply:
0929a64